etna/auto/auto.py

```diff
@@ -244,6 +249,83 @@ def __init__(
             metrics=metrics,
         )
         self.pool = pool
+        self._pool = self._make_pool(pool=pool, horizon=horizon)
+        self._pool_optuna: Optional[Optuna] = None
+
+        root_folder = self.experiment_folder if self.experiment_folder is not None else ""
+        self._pool_folder = f"{root_folder}/pool"
```

Comment:
If `root_folder` is None we'll get somenthing like `/pool`

Reply:
It was my intention. Is it bad. What name should we use in this case?

Reply:
I guess in case of saving artifacts like files - we would save it to `/pool` folder and it's a potential risky situation

Reply:
Do you suggest using `pool` instead of `/pool` if there is not `self.experiment_folder`? I'm ok with this.